packages/api/.eslintrc.json

```
@@ -0,0 +1,7 @@
+{
+    "extends": "@comet/eslint-config/nestjs",
+    "ignorePatterns": ["src/mikro-orm/migrations/**", "lib/**"],
```

```suggestion
    "ignorePatterns": ["lib/**"],
```

We don't ignore migrations in the Starter anymore, so we shouldn't here either

---

I removed this in 342f4bf10c40e957ca89d89de2e1877ed50a6938